ci-operator/jobs/openshift/kubernetes/openshift-kubernetes-master-presubmits.yaml

```diff
@@ -861,6 +861,7 @@ presubmits:
     optional: true
     path_alias: k8s.io/kubernetes
     rerun_command: /test e2e-azure
+    run_if_changed: ^(openshift-hack/images/hyperkube/.*|)$
```

Comment:
This is an "on rebase" trigger.  But can we catch Azure-touching changes more directly with something like `run_if_changed: azure`?  Seems like an unlikely substring for Azure-agnostic source.

Reply:
I don't think we have any way of triggering on all changes that have an impact on azure.